spec/book_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './spec_helper'
+
+describe Person do
+  context 'test book basic method' do
+    it 'test instance of book' do
+      @book = Book.new('Lord of ring', 'Tolkien')
+      expect(@book).to be_kind_of Book
+    end
+    it 'test book data' do
+      @book = Book.new('Lord of ring', 'Tolkien')
+      expect(@book.title).to eq 'Lord of ring'
+      expect(@book.author).to eq 'Tolkien'
+    end
+  end
+end
```

Comment:
Well done implementing this test so far. 

- [ ] Based on the project requirements, you are expected to test all methods inside of all your classes. IN this Book class, you have `add_rental` and `to_json` methods but you did not test those. Kindly add test for those methods